app/controllers/katello/api/v2/content_view_puppet_modules_controller.rb

```diff
@@ -9,7 +9,7 @@ class Api::V2::ContentViewPuppetModulesController < Api::V2::ApiController
     param :content_view_id, :number, :desc => N_("content view identifier"), :required => true
     param :name, String, :desc => N_("name of the puppet module")
     param :author, String, :desc => N_("author of the puppet module")
-    param :uuid, String, :desc => N_("uuid of the puppet module"), :deprecated => true
+    param :uuid, String, :desc => N_("uuid of the puppet module.  Will be removed in Katello 4.1."), :deprecated => true
```

Comment:
When is Puppet itself being removed?  Should we just delete these "uuid" deprecations and wait til then?

Reply:
@jlsherrill do we know if/when we will utterly remove puppet support (not just content)?

Reply:
puppet content support is still in the code, but upgrading to katello 4.0 will remove all the pulp support, so this entire controller will be useless (and the UI will no longer even show up).

We just haven't gotten around to removing the code parts yet

Reply:
>  Should we just delete these "uuid" deprecations and wait til then?

^ Based on what @jlsherrill just said it seems like this is a good course of action for now.  Let me know if anyone thinks differently